### ISSUE TITLE
Fix Ci - dev charts stopped generating properly

### ISSUE THREAD

**Issue by sumo-drosiek** [SumoLogic/sumologic-kubernetes-collection#1055]
Do not push linux-amd64 and shellcheck to the gh-pages

https://travis-ci.com/github/SumoLogic/sumologic-kubernetes-collection/jobs/417912888

```
error: The following untracked working tree files would be overwritten by checkout:
	linux-amd64/LICENSE
	linux-amd64/README.md
	linux-amd64/tiller
	shellcheck-v0.7.1/LICENSE.txt
	shellcheck-v0.7.1/README.txt
	shellcheck-v0.7.1/shellcheck
```

**Comment by sumo-drosiek** [comment]
@perk-sumo Fixed by #1056. Leaving Issue as we need to clean-up gh-pages branch

**Comment by perk-sumo** [comment]
GH Pages have been cleaned up.
